docs/preview/continuous-patching/README.md

```diff
@@ -0,0 +1,477 @@
+
+
+## Prerequisites        
+
+- You can use the Azure Cloud Shell or a local installation of the Azure CLI with a minimum version of 2.15.0 or later. 
+- You have an existing Resource Group with an Azure Container Registry.
+- You have an Azure Container Registry with Tasks enabled (Tasks is not supported in the free tier of ACR) 
+
+## Key Concepts
+Because Continuous Patching in ACR creates a new image per patch, ACR relies on a tag convention to version and identify patched images. The two main approaches are incremental and floating.
+
+### Incremental Tagging
+How It Works
+
+Each new patch increments a numerical suffix (e.g., -1, -2, etc.) on the original tag. For instance, if the base image is python:3.11, the first patch creates python:3.11-1, and a second patch on that same base tag creates python:3.11-2.
+
+Special Suffix Rules
+
+- -1 to -999: These are considered patch tags.
+- -x where x > 999: These are not interpreted as patch tags; instead, that entire suffix is treated as part of the original tag. (Example: ubuntu:jammy-20240530 is considered an original tag, not a patched one.)
+This means if you push a new tag ending in -1 to -999 by accident, Continuous Patching will treat it like a patched image. We recommend you to avoid pushing tags that you want patched with the suffix -1 to -999.
+
+### Floating Tagging
+
+How it works
+
+A single mutable tag, -patched, will always reference the latest patched version of your image. For instance, if your base image tag is python:3.11, one patch creates python:3.11-patched. Subsequent patches will keep overwriting that same -patched tag with new digests.
+
+  
+![PatchingTimelineExample](./media/patching_timeline_example1.png)
+
+### Which Should I Use?
+
+Incremental (**default**): Great for environments where auditability and rollbacks are critical, since each new patch is clearly identified with an unique tag.
+
+Floating: Ideal if you prefer a single pointer to the latest patch for your CI/CD pipelines. Reduces complexity by removing the need to update references in downstream applications per patch, but sacrifices strict versioning, making it difficult to rollback. 
+
+
+## Installing the Continuous Patching Workflow
+
+Download the [private CLI extension](https://acrcssc.z5.web.core.windows.net/acrcssc-1.1.1rc6-py3-none-any.whl) for Continuous patching 
+
+Run the following command:
+```sh
+az extension add --source <path to the CLI extension>
+```
+
+## Enable the Continuous Patching Workflow
+To enable Continuous Patching, follow the series of steps below that outline the CLI process. These guidelines detail the lifecycle of a continuous patching workflow, from its creation to subsequent updates to eventual deletion. 
+1. Login to Azure CLI with az login
+```sh
+az login
+```
+2. Login to ACR 
+```sh
+az acr login -n <myRegistry>
+```
+3. Use your preferred text editor to create a file named continuouspatching.json. Linux example below
```

Comment:
Please change this to create the file with a `heredoc` command, which is the standard way to create files through a single terminal command in typical developer guides. Make sure `continuouspatching.json` is in an inline code block.

3. Run the following command to create a file named `continuouspatching.json`, which contains the Continuous Patching instructions.

```sh
cat <<EOF > continuouspatching.json
{
    "version": "v1",
    "tag-convention" : "<incremental|floating>",
    "repositories": [{
        "repository": "<Repository Name>",
        "tags": ["<comma-separated-tags>"],   
        "enabled": <true|false>
    }] 
}
EOF
```


![image](https://github.com/user-attachments/assets/eda9e031-0816-49d4-b7d9-05ced0430b78)